frontend/public/components/utils/file-input.tsx

```diff
@@ -3,14 +3,26 @@ import * as classNames from 'classnames';
 import { NativeTypes } from 'react-dnd-html5-backend';
 import { DropTarget } from 'react-dnd';
 import { ConnectDropTarget, DropTargetMonitor } from 'react-dnd/lib/interfaces';
+import { Alert } from '@patternfly/react-core';
+import i18next from 'i18next';
+/* eslint-disable import/named */
+import { WithTranslation, withTranslation } from 'react-i18next';
 
 import withDragDropContext from './drag-drop-context';
 
 // Maximal file size, in bytes, that user can upload
 const maxFileUploadSize = 4000000;
-const fileSizeErrorMsg = 'Maximum file size exceeded. File limit is 4MB.';
+const fileSizeErrorMsg = i18next.t('utils~Maximum file size exceeded. File limit is 4MB.');
```

Comment:
Don't use `i18next.t` outside of a component or function. This message should be moved into the component.

Reply:
done thanks